src/main/java/com/sigmaukraine/messenger/validation/UserValidator.java

```diff
@@ -0,0 +1,73 @@
+package com.sigmaukraine.messenger.validation;
+
+import com.sigmaukraine.messenger.domain.User;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.Errors;
+import org.springframework.validation.ValidationUtils;
+import org.springframework.validation.Validator;
+
+@Component
+public class UserValidator implements Validator{
+
+    @Override
+    public boolean supports(Class<?> aClass) {
+        return User.class.isAssignableFrom(aClass);
+    }
+
+    /**
+     * Validates user's data. Login is required. Login can not be more then 20 symbols.
+     * FirstName can not be more then 30 symbols. LastName can not be more then 30 symbols.
```

Comment:
such comments are bad because you need to always keep them up to date with the code. code is self-explanatory enough without comments